### DISCUSSION TITLE
0.48.0 does try to build code and apparently it fails

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/panda#3004] [General]
### Description

Just tried to update to version 0.48.0 and it doesnt work. it starts a building process after installation and it fails

npm error $ tsup
npm error Cannot find src/index.ts,src/colors/*.ts,src/utils/index.ts
npm error error: script "build" exited with code 1

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Panda CSS Version

0.48.0

### Operating System

- [ ] macOS
- [x] Windows
- [ ] Linux

### Additional Information

I am using NX

**Comment by xeinebiu** [comment]
[not_working_example.zip](https://github.com/user-attachments/files/17966848/not_working_example.zip)

Here is the not working project, which does work perfecty fine in other versions except the last.

**Comment by anubra266** [comment]
@xeinebiu It's exactly what it means. We convert issues to discussions when we think it's not an issue with pandacss, and instead it needs further discussion to assist you in solving the problem.

In your case, you say the new version doesn't work. But it definitely does. 
But to reproduce whatever specific issue you're facing, you'd need to provide reproduction steps, ideally an online reproduction.

That said; I took the *risk* of __downloading your zip to my local system__, found a `start` script which ran successfully after install.
No idea what other issue you're facing, or how to reproduce it.

![CleanShot 2024-11-30 at 13 27 10@2x](https://github.com/user-attachments/assets/e87d9b1b-b307-44cc-9a33-c72e1ca170b6)

**Reply by xeinebiu** [reply]
> @xeinebiu It's exactly what it means. We convert issues to discussions when we think it's not an issue with pandacss, and instead it needs further discussion to assist you in solving the problem.
> 
> In your case, you say the new version doesn't work. But it definitely does. But to reproduce whatever specific issue you're facing, you'd need to provide reproduction steps, ideally an online reproduction.
> 
> That said; I took the _risk_ of **downloading your zip to my local system**, found a `start` script which ran successfully after install. No idea what other issue you're facing, or how to reproduce it.
> 
> ![CleanShot 2024-11-30 at 13 27 10@2x](https://private-user-images.githubusercontent.com/30869823/391289415-e87d9b1b-b307-44cc-9a33-c72e1ca170b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODMxOTAsIm5iZiI6MTczMzA4Mjg5MCwicGF0aCI6Ii8zMDg2OTgyMy8zOTEyODk0MTUtZTg3ZDliMWItYjMwNy00NGNjLTlhMzMtYzcyZTFjYTE3MGI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDE5NTQ1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczNWJlYjcwYzkxOWZhZDMzNjYyMDY1YTljODk5ZmZlMDVlYmVhZjU1ZGRjMzcyMjRkZmYwOTIzYzI0ZjllYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.II3JUMmhyQqp00KDBIvngnp8cnWNoqzvmftVtBMmtfE)

Should be related to Windows OS then. One time it complains "bun" is not found, then I do install bun even tho i dont need it, then it complains tsup is missing, i do install it as well. Then it starts compiling but it complains that src/** is not found while compiling.

I have posted some errror logs before
npm error $ tsup
npm error Cannot find src/index.ts,src/colors/*.ts,src/utils/index.ts
npm error error: script "build" exited with code 1

About the discussion, I am not sure If I 100% agree with you, because this "issue" clearly it "works on my machine" then let me convert it to a discussion, but I disagree because I have tried and spent more than 1h to fix the issue and it clearly is something wrong on version 0.48.0 because I do not face any issue from above on version 0.47.0

How is this not an issue?


Also seeing on the image, I see panda-css 0.46.1 while I complained on version 0.48.0
This should be an error on my side as apparently on the example I just updated ark to 4.4.4 but not pandacss dev
![image](https://github.com/user-attachments/assets/e8a957b6-6fa2-4da0-968a-469670b0348c)

Please try to update it to version 0.48.0, otherwise if it still works, then it is a Windows issue.

**Reply by xeinebiu** [reply]
Now after I changed the pandacss to 0.48.0 I receive

PS D:\repo\parkui> npm i
npm error code 1
npm error path D:\repo\parkui\node_modules\@park-ui\panda-preset
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c bun run build
npm error $ tsup
npm error Error: Cannot find module 'typescript'
npm error Require stack:
npm error - C:\ProgramData\nvm\v22.11.0\node_modules\tsup\dist\index.js
npm error - C:\ProgramData\nvm\v22.11.0\node_modules\tsup\dist\chunk-SNM7IVOJ.js
npm error - C:\ProgramData\nvm\v22.11.0\node_modules\tsup\dist\cli-default.js
npm error     at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
npm error     at Function._load (node:internal/modules/cjs/loader:1075:27)
npm error     at TracingChannel.traceSync (node:diagnostics_channel:315:14)
npm error     at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
npm error     at Module.require (node:internal/modules/cjs/loader:1340:12)
npm error     at require (node:internal/modules/helpers:141:16)
npm error     at Object.<anonymous> (C:\ProgramData\nvm\v22.11.0\node_modules\tsup\dist\index.js:1023:19)
npm error     at Module._compile (node:internal/modules/cjs/loader:1546:14)
npm error     at Object..js (node:internal/modules/cjs/loader:1689:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1318:32)
npm error error: script "build" exited with code 1

https://github.com/user-attachments/assets/01bde853-576d-427e-b09b-93b0e5c007dc


This is after installing typescript, bun, tsup globally

https://github.com/user-attachments/assets/7c98a9c6-f34b-4914-947d-74c74ddeb079

I did inspect the installed package files, and there is no "src" folder
![image](https://github.com/user-attachments/assets/ab65e856-3775-4945-99f6-20d32b9b2df7)

**Comment by anubra266** [comment]
@xeinebiu  could not reproduce

![CleanShot 2024-12-01 at 12 08 01@2x](https://github.com/user-attachments/assets/8ca36485-e72a-4011-b2f1-5c5c22edbc4d)

Also tried a build; runs as expected:

![CleanShot 2024-12-01 at 12 10 39@2x](https://github.com/user-attachments/assets/2449809c-be76-4da9-81dd-305339623ae5)

Also could you try removing node modules and reinstalling, dependencies in your project could be corrupted already.

**Reply by xeinebiu** [reply]
> @xeinebiu could not reproduce
> 
> ![CleanShot 2024-12-01 at 12 08 01@2x](https://private-user-images.githubusercontent.com/30869823/391360065-8ca36485-e72a-4011-b2f1-5c5c22edbc4d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODQxODEsIm5iZiI6MTczMzA4Mzg4MSwicGF0aCI6Ii8zMDg2OTgyMy8zOTEzNjAwNjUtOGNhMzY0ODUtZTcyYS00MDExLWIyZjEtNWM1YzIyZWRiYzRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDIwMTEyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3YzNmOGQ2NmJmYzA1YmQzMzY3ZjA5NjM2MjFmNjI2ZDRlOWY3YTEwMGRmMjI2N2EyNWM2Y2NkN2M0ZDMwZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.50rzC5ySL9w27dN7u2H8nritSVBvhafwk0-3CIz4o_s)
> 
> Also tried a build; runs as expected:
> 
> ![CleanShot 2024-12-01 at 12 10 39@2x](https://private-user-images.githubusercontent.com/30869823/391360239-2449809c-be76-4da9-81dd-305339623ae5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMwODQxODEsIm5iZiI6MTczMzA4Mzg4MSwicGF0aCI6Ii8zMDg2OTgyMy8zOTEzNjAyMzktMjQ0OTgwOWMtYmU3Ni00ZGE5LTgxZGQtMzA1MzM5NjIzYWU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAxVDIwMTEyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg0ODcyYjQ4YjRiYmYzZDliMjMwYTViMDhhZDk2ODAwOGRiMGU4ZDU4MGQ3NTA2ZjRhN2YyY2JkZDE1ZGI3NmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.imO7AUAbITe0qyS_lwQ-Dk-Z8sd5g1KjFbax_yUH3Yc)
> 
> Also could you try removing node modules and reinstalling, dependencies in your project could be corrupted already.


https://github.com/user-attachments/assets/15070ba6-6599-4139-b802-50b0f9140200

I found the issue is related to PARK-UI and not PandaCSS, my bad and I do apologise for the confusion. You can see on the video version 0.43.1 of ParkUI does cause the issue.

**Reply by xeinebiu** [reply]
A fresh installation, still PARKUI fails, but not an issue of PandaCSS it self.

https://github.com/user-attachments/assets/08166d00-e4e3-4127-b395-ce8e7e6baa85

@cschroeter Can you please help on the investigation on this regard?

**Reply by anubra266** [reply]
No worries. Glad to help
